pilot/pkg/config/kube/crdclient/types.gen.go

```diff
@@ -47,56 +48,67 @@ func create(ic versionedclient.Interface, sc serviceapisclient.Interface, cfg co
 		return ic.NetworkingV1alpha3().DestinationRules(cfg.Namespace).Create(context.TODO(), &clientnetworkingv1alpha3.DestinationRule{
 			ObjectMeta: objMeta,
 			Spec:       *(cfg.Spec.(*networkingv1alpha3.DestinationRule)),
+			Status:     *(cfg.Status.(*v1alpha1.IstioStatus)),
```

Comment:
I think spec will be empty for most use cases? so are we going to get nil pointer dereferences here?

Reply:
do you mean status?

spec was there before

Reply:
yes sorry